impl/src/test/java/org/ehcache/impl/internal/store/offheap/OffHeapStoreTest.java

```diff
@@ -121,7 +121,7 @@ public void testRank() throws Exception {
   private void assertRank(final Store.Provider provider, final int expectedRank, final ResourceType<?>... resources) {
     assertThat(provider.rank(
       new HashSet<>(Arrays.asList(resources)),
-        Collections.<ServiceConfiguration<?>>emptyList()),
+        Collections.<ServiceConfiguration<?, ?>>emptyList()),
```

Comment:
I believe you can elide the generic parameter here.

Reply:
Fixed.